nominees/covid-19-api.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "Covid-19 API",
+  "description": "This API can be utilised by all applications that need to write/retrieve Ethiopia Covid19 data. E.g. covid19.et project will use it to retrieve data for dashboard and write data",
+  "website": "https://ethiocovid19rt.com/projects",
+  "license": [
+    {
+      "spdx": "MIT",
+      "licenseURL": "https://github.com/africa-covid-19-response-toolkit/api/blob/master/LICENSE"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 3,
+      "evidenceURL": "https://ethiocovid19rt.com/about"
```

Comment:
@rachita19082 in as much as possible, try to have `evidenceURL` point to external blogs, articles rather than owner's website pages. It would be good to make the above update for this before we can have it approved and merged. Thanks.